requirements-dev.txt

```diff
@@ -135,8 +131,6 @@ ruamel-yaml==0.17.20
     # via ansible-lint
 ruamel-yaml-clib==0.2.6
     # via ruamel-yaml
-selinux==0.2.1
-    # via molecule-docker
```

Comment:
⚠️ This is going away because you are on macOS and not on Linux.

Reply:
Gah, it got me again 😓

Reply:
See https://github.com/Ableton/ansible-role-jenkins-swarm-client/pull/95/commits/5af9f15473acfb4cd8b3ae6f7c1621a5348ecd22.